File: Volo.Abp/Volo.Chat/src/Volo.Chat.Application.Contracts/Volo/Chat/Messages/ChatMessageDto.cs
using System;

namespace Volo.Chat.Messages
{
    public class ChatMessageDto
    {
        public string Message { get; set; }

        public DateTime MessageDate { get; set; }

        public bool IsRead { get; set; }

        public DateTime ReadDate { get; set; }

        public ChatMessageSide Side { get; set; }
    }
}

File: Volo.Abp/Volo.Chat/angular/projects/chat/src/lib/models/chat-message-dto.ts
import { ChatMessageSide } from '../enums/chat-message-side';

export class ChatMessageDto {
  message: string;
  messageDate: Date | string;
  isRead: boolean;
  readDate: Date | string;
  side: ChatMessageSide;

  constructor(initialValues: Partial<ChatMessageDto> = {}) {
    if (initialValues) {
      for (const key in initialValues) {
        if (initialValues.hasOwnProperty(key)) {
          this[key] = initialValues[key];
        }
      }
    }
  }
}
